packages/odata-common/src/request/odata-get-all-request-config.ts

```diff
@@ -23,9 +24,13 @@ export class ODataGetAllRequestConfig<
   top: number;
   skip: number;
   filter: Filterable<EntityT, DeSerializersT>;
-  orderBy: Orderable<EntityT>[];
+  orderBy: Orderable<EntityT, EntityApi<EntityBase>>[];
   selects: Selectable<EntityT, DeSerializersT>[];
-  expands: Expandable<EntityT, DeSerializersT>[];
+  expands: Expandable<
+    EntityT,
+    DeSerializersT,
+    EntityApi<EntityBase, DeSerializersT>
```

Comment:
Aren't the last parameters of Orderable defaults? would it make sense to remove them here to emphasize that they are "irrelevant"?